deltacat/compute/compactor_v2/utils/task_options.py

```diff
@@ -79,14 +78,7 @@ def get_task_options(
 
     # List of possible botocore exceptions are available at
     # https://github.com/boto/botocore/blob/develop/botocore/exceptions.py
-    task_opts["retry_exceptions"] = [
-        botocore.exceptions.ConnectionError,
-        botocore.exceptions.HTTPClientError,
-        ConnectionError,
-        TimeoutError,
-        DaftTransientError,
-        tenacity.RetryError,
-    ]
+    task_opts["retry_exceptions"] = RAY_TASK_RETRYABLE_ERROR_CODES
```

Comment:
We can just hard-code `RetryableError` here.

Reply:
Added TODO